HELP.md

```diff
@@ -0,0 +1,109 @@
+# Contract best practices
+
+## Init once
+
+Make sure you check that state doesn't exists in `init` using
+
+```rust
+assert!(!env::state_exists(), "The contract is already initialized");
```

Comment:
Should this become an argument on the `init` marco? e.g. `init(once)`?

Reply:
Or `#[init_once]` https://github.com/near/near-sdk-rs/issues/95